testapi.pm

```diff
@@ -396,6 +397,24 @@ sub wait_screen_change(&@) {
     return 0;
 }
 
+=head2 assert_screen_change
+
+  assert_screen_change { CODEREF [,$timeout] };
```

Comment:
This is actual incorrect syntax because the CODEREF is the {} - and the optinal timeout is after it. The same error is in the wait_screen_change reference